crates/sui-graphql-rpc/src/client/simple_client.rs

```diff
@@ -148,3 +152,29 @@ pub fn resolve_variables(
 
     Ok((type_defs, var_vals))
 }
+
+pub const fn is_valid_variable_name_char(c: char) -> bool {
```

Comment:
Any particular reason why `_` is allowed but not `-`?

Reply:
GraphQL does not allow hyphen in variable names.
Hyphen is usually reserved for subtraction in most languages.